admin-dev/themes/new-theme/scss/components/layout/_main_header.scss

```diff
@@ -30,6 +30,15 @@
     height: 100%;
     width: $size-navbar-width;
   }
+  #ajax_spinner {
```

Comment:
I think it will probably be better if you used a class name instead (so that you could have multiple spinners on one page).

In addition, please use dash (`-`) instead of underscore (`_`) for classes and IDs.